tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1204,15 +1117,14 @@ def test_deletes_after_kpods_down(self):
                                                      "operations. Found failures in READ: "
                                                      f"{fail_bkt_get} {event_bkt_get}"
                                                      f"or DI_CHECK: {fail_di_bkt} {event_di_bkt}")
-        LOGGER.info("Step 7: Successfully performed READs on the remaining %s buckets.", remain_bkt)
+        LOGGER.info("Step 4: Successfully performed READs on the remaining %s buckets.", remain_bkt)
 
         LOGGER.info("ENDED: Test to verify DELETEs after %s (K) data pods down.", self.kvalue)
 
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.skip(reason="Blocked until 'CORTX-27549' resolve")
```

Comment:
instead we can latest defect ID which we filed for VM down with UUID issue. cc: @ApurwaMohite